llvm/tools/llvm-dwarfdump/llvm-dwarfdump.cpp

```diff
@@ -295,6 +295,10 @@ static opt<ErrorDetailLevel> ErrorDetails(
            clEnumValN(BothDetailsAndSummary, "full",
                       "Display each error as well as a summary. [default]")),
     cat(DwarfDumpCategory));
+static opt<std::string> AggregationJsonFile(
+    "aggregate-output-file", cl::init(""),
+    cl::desc("Output JSON-formatted error summary to the specified file."),
```

Comment:
I think the documentation makes it unclear whether this is supposed to be used only in conjunction with `--verify`?

Reply:
Added better description.